lib/project-data.ts

```diff
@@ -30,7 +30,16 @@ export class ProjectData implements IProjectData {
 	public projectDir: string;
 	public platformsDir: string;
 	public projectFilePath: string;
-	public projectId: string;
+	public projectIdentifiers: IProjectIdentifier;
+	get projectId(): string {
+		this.warnProjectId();
+		return this.projectIdentifiers.ios;
```

Comment:
why does a common module `project-data.ts` return only an iOS project identifier?

Reply:
This is intended to let hooks and extension that use the old property to behave as before.
Before if the user changed the application identifier for Android he must use the gradle file and the rest of the code will still use the one inside the package.json (the one for iOS).

Reply:
maybe put a comment in the code, because it looks a bit strange returning only ios identifier in a common file

Reply:
Will do that.